mrp_subcontracting_bom_dual_use/models/mrp_production.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class MrpProduction(models.Model):
+    _inherit = "mrp.production"
+
+    bom_id = fields.Many2one(
```

Comment:
Comment why this is here, as it's not obvious:

```suggestion
    # Overwrite the field domain for adding subcontracting BoMs with the check
    bom_id = fields.Many2one(
```